### PR TITLE
Fixed: respect multiple pseudo classes for `webkit` pseudo elements in `selector-pseudo-class-no-unknown` rule.

### PR DESCRIPTION
**Pull request by evilebottnawi**
> Which issue, if any, is this issue related to?

#2810 

> Is there anything in the PR that needs further explanation?

No, it's self explanatory.

### CODE REVIEW

File: lib/rules/selector-pseudo-class-no-unknown/index.js [hunk 1]
```diff
@@ -84,9 +84,17 @@ const rule = function(actual, options) {
               return;
             }
 
-            if (pseudoNode.prev()) {
+            const firstSelectorNode =
+              pseudoNode.parent && pseudoNode.parent.nodes
```
Comment (evilebottnawi) [inline lib/rules/selector-pseudo-class-no-unknown/index.js]: Should we check `parent` here? Be good to find use case where `pseudoNode` doesn't contain `parent` (if this use case exists).

File: lib/rules/selector-pseudo-class-no-unknown/__tests__/index.js [hunk 2]
```diff
@@ -90,6 +90,10 @@ testRule(rule, {
     {
       code: "html { --custom-property-set: {} }",
       description: "custom property set in selector"
+    },
+    {
+      code: "::-webkit-scrollbar-button:horizontal:decrement {}",
+      description: "webkit scrollbar multiple pseudo classes"
```
Comment (ntwb) [inline lib/rules/selector-pseudo-class-no-unknown/__tests__/index.js]: Let's use `description: "webkit scrollbar chained pseudo classes"` as chained is the correct terminology I believe

File: lib/rules/selector-pseudo-class-no-unknown/index.js (hunk 1 above)
Comment (ntwb) [reply lib/rules/selector-pseudo-class-no-unknown/index.js]: cc @jeddy3 thoughts?

Comment (ntwb) [reply lib/rules/selector-pseudo-class-no-unknown/index.js]: We can probably sneak this into 8.1.0 if no other chanfes to the PR are required

File: lib/rules/selector-pseudo-class-no-unknown/__tests__/index.js (hunk 2 above)
Comment (ntwb) [reply lib/rules/selector-pseudo-class-no-unknown/__tests__/index.js]: I made this change in https://github.com/stylelint/stylelint/pull/2813/commits/21cf021454fe9a07c045513c2193398117daa5ee

**Comment by evilebottnawi** [comment]
@ntwb thanks, no time now, sorry :disappointed:

File: lib/rules/selector-pseudo-class-no-unknown/index.js (hunk 1 above)
Comment (jeddy3) [reply lib/rules/selector-pseudo-class-no-unknown/index.js]: > Should we check parent here?

Sure, no harm in guarding against it.

**Comment by jeddy3** [comment]
This PR just requires a rebase and a small tweak, then I think we can get it in.

**Comment by evilebottnawi** [comment]
@jeddy3 done

**Comment by ntwb** [comment]
Changelog via https://github.com/stylelint/stylelint/commit/20b200979a717526a73d1b094b9d93af3d2bb97d
> -   Fixed: `selector-pseudo-class-no-unknown` false positives when using chained pseudo-classes ([#2810](https://github.com/stylelint/stylelint/issues/2810)).

**Pull request merged**